src/components/DescriptionCocktail.jsx

```diff
@@ -0,0 +1,85 @@
+import { useState, useEffect } from 'react';
+import { Box, UnorderedList, ListItem, Spacer, Image, Stack, StackDivider, Heading, Text, Wrap, WrapItem } from '@chakra-ui/react';
+import { useParams } from 'react-router-dom';
+
+const DescriptionCocktail = () => {
+  const [cocktail, setCocktail] = useState(null);
+  const params = useParams();
+  let { drinkId } = useParams();
```

Comment:
`const` car jamais réassigné
tu peux utiliser `params` directement, pas besoin de re-call le hook `useParams()`
```suggestion
  const { drinkId } = params;
```